x/wasm-storage/keeper/keeper.go

```diff
@@ -46,6 +34,7 @@ type Keeper struct {
 	Schema                collections.Schema
 	DataRequestWasm       collections.Map[[]byte, types.Wasm]
 	OverlayWasm           collections.Map[[]byte, types.Wasm]
+	WasmExp               collections.KeySet[collections.Pair[int64, []byte]]
```

Comment:
`WasmExpirations` ?

Reply:
This is not a queue. On gurantee of order.